src/huntsman/pocs/camera/zwo.py

```diff
@@ -166,6 +165,11 @@ def connect(self):
         self._control_info = Camera._driver.get_control_caps(self._handle)
         self._info['control_info'] = self._control_info  # control info accessible via properties
         Camera._driver.disable_dark_subtract(self._handle)
+
+        if self._gain is not None:
+            self.gain = self._gain
+        self.image_type = self._image_type
```

Comment:
It's theoretically possible to get an `AttributeError` here. If `image_type` wasn't passed to `__init__` and the camera model doesn't support `RAW16` image type then `self._image_type` will not have been set. It can't happen with the type of cameras Huntsman has, but better to avoid the possibility anyway.

```suggestion
        with suppress(AttributeError):
            self.image_type = self._image_type
```